datatables-parent/datatables/src/main/java/org/wicketstuff/datatables/DemoDatatable.java

```diff
@@ -53,7 +54,7 @@ private String getJUITheme()
 
 	private void renderBasicJS(IHeaderResponse response)
 	{
-		response.render(JavaScriptHeaderItem.forReference(new PackageResourceReference(DemoDatatable.class, "media/js/jquery.js")));
+		response.render(JavaScriptHeaderItem.forReference(JQueryResourceReference.getV3()));
```

Comment:
Would it be better to use `Application.get().getJavaScriptLibrarySettings().getJQueryReference()` ?

this way user defined JQuery will be honored ...

Reply:
Thank you @solomax Will do the changes